ufo/static/locales/en/messages.json

```diff
@@ -96,5 +96,10 @@
   "saveMultipleUsersButton": "Add Users",
   "saveIndividualUserButton": "Add User",
   "versionText": "Version: ",
-  "versionUpdateText": "A newer version is available on Github: "
+  "versionUpdateText": "A newer version is available on Github: ",
+  "emailValidationError": "Please supply a valid email address.",
+  "keyLookupError": "Please supply a valid email address or unique id.",
+  "publicKeyError": "Please supply a valid public key in a format such as 'ssh-rsa AAAA...' (no email address afterwards).",
```

Comment:
Thanks for adding a note about the email!  But can we also add an extra word "comment", since someone can actually do that?  Such as: ("no email address or comments afterwards)"